src/app/beer_garden/api/http/handlers/v1/garden.py

```diff
@@ -152,9 +146,10 @@ async def patch(self, garden_name):
                 )
             elif operation == "config":
                 garden_to_update = self.validate_or_raise(
-                    GardenSchema, op.value, method="load"
+                    GardenSchema, op.value, from_string=False
```

Comment:
If a bad garden JSON is passed, it will blow up here.

Reply:
A few things that came mind when looking at this again:

* One thing worth noting, which is different from your original, separate endpoint version, is that this takes a complete garden definition, rather than just the `connection_params` portion of the config,  That seems okay based on the way `GardenSchema` is currently defined, since everything but the name is considered optional.
*  The name being required when it's already provided in the path isn't great, so you might want to look at populating it based on the path parameter.
*  Name really shouldn't be a thing that you can change.  I think this endpoint will let you do it, but doing so will break a ton of stuff.  That's an existing problem so you don't have to fix it here, I just mention it because I don't want you to go to any trouble to ensure that changing the name could theoretically be done here.  I think you just take the name from the path parameter and not allow it to be set separately here.